doc/nrf/protocols/matter/getting_started/advanced_kconfigs.rst

```diff
@@ -102,3 +102,30 @@ To read more about the FFS technology and its compatibility with Matter, see the
 
 * `Matter Simple Setup for Wi-Fi Overview`_
 * `Matter Simple Setup for Thread Overview`_
+
+Reaction to the last Matter fabric removal
+==========================================
+
+When a Matter device is commissioned by the Matter controller it starts belonging to a specific Matter fabric.
+To ensure interoperability of multiple applications and ecosystems the device can join multiple Matter fabrics created by multiple ecosystems using the multi-fabric feature.
+As a result, the device can belong to multiple fabrics and can be managed by multiple Matter controllers.
+To remove the device from a specific fabric a Matter controller sends the leave request to it and then removes all information collected from the device.
+The device receives the leave request and removes all information about the leaving fabric.
+When the device leaves the last fabric there are several reactions to that to be set.
+
+To enable one of the reactions to the last fabric removal choose and set the following configuration option:
+
+* :kconfig:option:`CONFIG_CHIP_LAST_FABRIC_REMOVED_NONE` to ``y`` to do not react to the last fabric removal.
+  The device will leave all saved data and network credentials and not reboot.
+* :kconfig:option:`CONFIG_CHIP_LAST_FABRIC_REMOVED_ERASE_ONLY` to ``y`` to remove all saved network credentials.
+  The device will remove all saved network credentials and leave application-specific non-volatile data.
+* :kconfig:option:`CONFIG_CHIP_LAST_FABRIC_REMOVED_ERASE_AND_PAIRING_START` to ``y`` to remove all saved network credentials and start Bluetooth LE advertising.
+  The device will remove all saved network credentials, leave application-specific non-volatile data, and start advertising Bluetooth LE Matter service.
+  After that, it will be ready for the new commissioning to Matter over Bluetooth LE.
```

Comment:
```suggestion
* :kconfig:option:`CONFIG_CHIP_LAST_FABRIC_REMOVED_ERASE_AND_PAIRING_START` - Remove all saved network credentials and start Bluetooth LE advertising.
  The device will remove all saved network credentials, keep application-specific non-volatile data, and start advertising Bluetooth LE Matter service.
  After that, it will be ready for commissioning to Matter over Bluetooth LE.
```